### PR TITLE
Port niceupdate to python3

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Retain compatibility with python2.

### CODE REVIEW

**Comment by a1exsh** [comment]
:+1:

File: niceupdate/niceupdate.py [hunk 1]
```diff
@@ -11,7 +11,9 @@
 import logging
 import termios
 import select
-import Queue
+import six
+from six.moves import reduce
```
Comment (avaczi) [inline niceupdate/niceupdate.py]: where is this used?

Comment (alexeyklyukin) [inline niceupdate/niceupdate.py]: the six.PY2 checks whether the interpreter is v2 or v3+.

Comment (avaczi) [inline niceupdate/niceupdate.py]: I mean the `reduce`

Comment (alexeyklyukin) [inline niceupdate/niceupdate.py]: It's used in the original niceupdate code, not in the diff: https://github.com/zalando/acid-tools/blob/efeccc7427fc29494b66fb00bba823380b23078f/niceupdate/niceupdate.py#L745

**Comment by a1exsh** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**